guides/common/modules/proc_viewing-multiple-content-view-environment-activation-keys.adoc

```diff
@@ -0,0 +1,36 @@
+[id="viewing-multiple-content-view-environment-activation-keys"]
+= Viewing multiple content view environment activation keys
+
+Use this procedure to view activation keys with multiple content view environments, or multi-environment activation keys, in the {ProjectWebUI} and Hammer CLI.
+
+.Prerequisites
+* Ensure you have activation keys created.
+For more information, see xref:Creating_an_Activation_Key_{context}[Creating an activation key].
```

Comment:
```suggestion
For more information, see xref:Creating_an_Activation_Key_{context}[].
```

suggestion: drop title. xrefs do not need titles. Omitting them makes it easier to maintain the docs.